lib/crowi/index.js

```diff
@@ -336,5 +344,24 @@ Crowi.prototype.exitOnError = function(err) {
   process.exit(1);
 };
 
+Crowi.prototype.getParsedVersion = function(version) {
+  version = version || this.version;
+
+  var major = 0;
+  var minor = 0;
+  var revision = 0;
+
+  if (version.match(/^(\d+)\.(\d+)\.(\d+)$/)) {
+    major    = +(RegExp.$1);
+    minor    = +(RegExp.$2);
+    revision = +(RegExp.$3);
```

Comment:
In my view, this method is used as both statically and dynamically. (used by es util class with es version number (static), used by the Crowi object with `this.version`)
IMO, the version number parser should not be the member function of Crowi class but be separated as an utility function, and this function just use the function to parse the versioning number `this.version`.